test/remote/gateways/remote_securion_pay_test.rb

```diff
@@ -0,0 +1,341 @@
+require 'test_helper'
+
+class RemoteSecurionPayTest < Test::Unit::TestCase
+  CHARGE_ID_REGEX = /char_[a-zA-Z\d]+/
+  TOKEN_ID_REGEX  = /tok_[a-zA-Z\d]+/
+
+  def setup
+    @gateway = SecurionPayGateway.new(fixtures(:securion_pay))
+
+    @amount = 2000
+    @refund_amount = 300
+    @credit_card = credit_card('4242424242424242')
+    @declined_card = credit_card('4916018475814056')
+    @new_credit_card = credit_card('4012888888881881')
+    @invalid_token = 'tok_invalid'
+
+    @options = {
+      description: 'ActiveMerchant test charge',
+      email: 'foo@example.com'
+    }
+  end
+
+  def test_dump_transcript
+    skip("Transcript scrubbing for this gateway has been tested.")
+    dump_transcript_and_fail(@gateway, @amount, @credit_card, @options)
+  end
+
+  def test_transcript_scrubbing
+    transcript = capture_transcript(@gateway) do
+      @gateway.purchase(@amount, @credit_card, @options)
+    end
+    transcript = @gateway.scrub(transcript)
+    assert_scrubbed(@credit_card.number, transcript)
+    assert_scrubbed(@credit_card.verification_value, transcript)
+    assert_scrubbed(@gateway.options[:secret_key], transcript)
+  end
+
+  def test_successful_purchase
+    assert response = @gateway.purchase(@amount, @credit_card, @options)
+    assert_success response
+    assert_equal "charge", response.params["objectType"]
```

Comment:
Tests should only rarely be checking `Response#params`. Rather, check the common attributes on `Response` - `authorization`, `message`, `avs_code`, etc.